### PR TITLE
redid all my changes in development and fixed the suggestions

### PR DESCRIPTION
**Pull request by swilcox21**

### CODE REVIEW

File: src/app/views/admissions/student-form/student-utils/AddBulkToCohort.jsx [hunk 1]
```diff
@@ -2,16 +2,19 @@ import React, { useState, useEffect, useMemo } from 'react';
...
 } from '@material-ui/core';
+import PropTypes from 'prop-types';
 import { AsyncAutocomplete } from '../../../../components/Autocomplete';
 import bc from '../../../../services/breathecode';
 
+const propTypes = {
+  classes: PropTypes.string.isRequired,
+  selectedRows: PropTypes.string.isRequired,
+  user: PropTypes.string.isRequired,
+  items: PropTypes.string.isRequired,
```
Comment (ArielCalisaya) [inline src/app/views/admissions/student-form/student-utils/AddBulkToCohort.jsx:15]: `items` is an array
expected:
```js
items: PropTypes.arrayOf(PropTypes.array).isRequired,
```

File: src/app/views/admissions/student-form/student-utils/ProfileForm.jsx [hunk 2]
```diff
@@ -1,16 +1,16 @@
...
+import { Grid, TextField, Button } from '@material-ui/core';
 import { useHistory } from 'react-router-dom';
-import bc from 'app/services/breathecode';
+import PropTypes from 'prop-types';
+import bc from '../../../../services/breathecode';
 import axios from '../../../../../axios';
 import { AsyncAutocomplete } from '../../../../components/Autocomplete';
 
+const propTypes = {
+  initialValues: PropTypes.string.isRequired,
```
Comment (ArielCalisaya) [inline src/app/views/admissions/student-form/student-utils/ProfileForm.jsx:11]: `initialValues` probably is an object change ~~`string`~~ to `object`

expected:
```js
initialValues: PropTypes.objectOf(PropTypes.object).isRequired,
```

File: src/app/views/certificates/ResponseDialog.jsx [hunk 3]
```diff
@@ -6,10 +6,27 @@ import DialogActions from '@material-ui/core/DialogActions';
...
 import DialogContentText from '@material-ui/core/DialogContentText';
 import DialogTitle from '@material-ui/core/DialogTitle';
+import PropTypes from 'prop-types';
 import CertificatesResult from './certificates-utils/CertificatesResults';
+// needs multiple differnt prop validations for same variable??
+const propTypes = {
+  openDialog: PropTypes.number.isRequired,
+  setOpenDialog: PropTypes.string.isRequired,
+  responseData: PropTypes.string.isRequired,
+  isLoading: PropTypes.string.isRequired,
```
Comment (ArielCalisaya) [inline src/app/views/certificates/ResponseDialog.jsx:16]: change ~~`string`~~ to `boolean`

File: src/app/views/certificates/certificates-utils/CertificateCustomToolbar.jsx [hunk 4]
```diff
@@ -1,8 +1,17 @@
...
 import { Tooltip, IconButton } from '@material-ui/core';
+import PropTypes from 'prop-types';
+import bc from '../../../services/breathecode';
+
+const propTypes = {
+  classes: PropTypes.number.isRequired,
+  selectedRows: PropTypes.string.isRequired,
+  items: PropTypes.string.isRequired,
+  loadData: PropTypes.string.isRequired,
+  setSelectedRows: PropTypes.string.isRequired,
```
Comment (ArielCalisaya) [inline src/app/views/certificates/certificates-utils/CertificateCustomToolbar.jsx:13]: change ~~`string`~~ to `array`

expected:
```js
setSelectedRows: PropTypes.arrayOf(PropTypes.array).isRequired,
```

File: src/app/views/certificates/certificates-utils/CertificateCustomToolbar.jsx [hunk 5]
```diff
@@ -1,8 +1,17 @@
...
 import PostAddIcon from '@material-ui/icons/PostAdd';
 import { withStyles } from '@material-ui/core/styles';
-import bc from 'app/services/breathecode';
 import { Tooltip, IconButton } from '@material-ui/core';
+import PropTypes from 'prop-types';
+import bc from '../../../services/breathecode';
+
+const propTypes = {
+  classes: PropTypes.number.isRequired,
+  selectedRows: PropTypes.string.isRequired,
```
Comment (ArielCalisaya) [inline src/app/views/certificates/certificates-utils/CertificateCustomToolbar.jsx:10]: change ~~`string`~~ to `array`
```js
selectedRows: PropTypes.arrayOf(PropTypes.array).isRequired,
```

**Review by ArielCalisaya — CHANGES_REQUESTED** [review]

**Review by ArielCalisaya — APPROVED** [review]

**Review by ArielCalisaya — APPROVED** [review]

**Review by alesanchezr — APPROVED** [review]

**Pull request merged**